uktrade/data-hub-frontend
# Investment projects refinements (new dashboard)

## Description of change

Lots of styling refinements around the new dashboard. See each commit for details.

## Test instructions

You will need to configure a few things to view the new dashboard, see PR #3248 for more details.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
